azext_edge/edge/providers/support/base.py

```diff
@@ -47,10 +47,10 @@ def process_crd(group: str, version: str, kind: str, plural: str, api_moniker: s
 
 def process_v1_pods(
     resource_api: EdgeResourceApi,
-    label_selector=None,
+    capture_previous_logs: bool = True,
+    include_metrics: bool = False,
     since_seconds: int = 60 * 60 * 24,
-    include_metrics=False,
-    capture_previous_logs=True,
+    label_selector: Optional[str] = None,
```

Comment:
nit: be careful when modifying positions of arguments since it could affect anything that calls it without specifying the exact param

Reply:
yeah but luckily all the calls have passed with param name!